src/components/collection/DataPreview/TableView.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+    Grid,
+    Paper,
+    SxProps,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+    Theme,
+} from '@mui/material';
+import Error from 'components/shared/Error';
+import { tableAlternateRowsSx } from 'context/Theme';
+import { useJournalData } from 'hooks/useJournalData';
+import { LiveSpecsQuery_spec } from 'hooks/useLiveSpecs';
+import { useMemo } from 'react';
+
+interface PreviewTableModeProps {
+    spec: LiveSpecsQuery_spec;
+    journalData: ReturnType<typeof useJournalData>;
+}
+
+const heightSx: SxProps<Theme> = {
+    minHeight: 350,
+    maxHeight: 350,
+};
+
+function TableView({
+    journalData: { data, error },
+    spec,
+}: PreviewTableModeProps) {
+    const specEntries = useMemo(
+        // TODO (typing) we need to fix typing
+        // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+        () => Object.entries(spec?.spec?.schema?.properties ?? {}),
```

Comment:
Many schemas won't have a top level properties that's useful for this. For example, you may use an allOf to compose multiple schemas, which is actually how we handle composing in the `_meta` schemas. It might be good to replace this schema iteration with a call to fetch projections, once that's supported. All good for now, though, since the JSON view is the default. Just sharing my thoughts on it.

Reply:
So yeah this was why I was asking about how to get the projections for a collection. "How hard would it be" to implement "GET projections"? Good next task for me?